wuzz.go

```diff
@@ -1009,25 +1018,27 @@ func (a *App) ParseArgs(g *gocui.Gui, args []string) error {
 			arg_index += 1
 			header := args[arg_index]
 			fmt.Fprintf(vheader, "%v\n", header)
-		case "-d", "--data":
+		case "-d", "--data", "--data-binary":
 			if arg_index == args_len-1 {
-				return errors.New("No POST/PUT value specified")
+				return errors.New("No POST/PUT/PATCH value specified")
 			}
 
-			vmethod, _ := g.View(REQUEST_METHOD_VIEW)
-			setViewTextAndCursor(vmethod, http.MethodPost)
-
 			arg_index += 1
-			add_content_type = true
+			set_data = true
+			set_binary_data = arg == "--data-binary"
 
-			data, _ := url.QueryUnescape(args[arg_index])
+			data := args[arg_index]
+			if !set_binary_data {
+				data, _ = url.QueryUnescape(data)
+			}
```

Comment:
The "binary" comes from the existing cURL option "--data-binary".  Since one feature of wuzz seems to be its compatibility with cURL options, I'd think keeping the same terminology would be less confusing.  That being said, cURL does have a "--data-raw" option as well, however Chrome (at least with the requests I've tested with) doesn't generate it for JSON bodies.

At least in the case of cURL, the data might actually be binary since the option supports reading the data from stdin if "--data-binary @-" is given.  For example `cat some_binary_file.bin | curl -X POST --data-binary @- http://httpbin.org/post`.